From 36f5fcc5e1ff0bf3604ef957248807ee640e7854 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Thu, 7 Jul 2016 13:57:47 +0300
Subject: [PATCH 0340/1345] pp2: remove ptp regs from gop110hw

commit  4fa9d65080dd6f3f2478cf4e657b999dd5e50e84 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

PTP/TAI hw units are handled separately and have own
reg-hw.h files.
This is also syncs to the working pp3 ptp code-tree

Change-Id: I084b50101232e5cb27a17ecc0d4baded26b55bc4
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30999
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../ethernet/marvell/mvpp2x/mv_gop110_hw_type.h    |  239 --------------------
 1 file changed, 239 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw_type.h b/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw_type.h
index f26695e..0ba4578 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw_type.h
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_gop110_hw_type.h
@@ -1847,245 +1847,6 @@
 #define MV_MIB_COLLISION			0x78
 #define MV_MIB_LATE_COLLISION			0x7c
 
-/*************/
-/*   PTP  REGS    */
-/*************/
-
-/* Ptp General Control */
-#define MV_PTP_GENERAL_CTRL_REG					(0x0808)
-#define MV_PTP_GENERAL_CTRL_PTP_UNIT_ENABLE_OFFS		0
-#define MV_PTP_GENERAL_CTRL_PTP_UNIT_ENABLE_MASK    \
-		(0x00000001 << MV_PTP_GENERAL_CTRL_PTP_UNIT_ENABLE_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_PTP_RESET_OFFS		1
-#define MV_PTP_GENERAL_CTRL_PTP_RESET_MASK    \
-		(0x00000001 << MV_PTP_GENERAL_CTRL_PTP_RESET_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_INTERFACE_WIDTH_SELECT_OFFS		2
-#define MV_PTP_GENERAL_CTRL_INTERFACE_WIDTH_SELECT_MASK    \
-		(0x00000003 << MV_PTP_GENERAL_CTRL_INTERFACE_WIDTH_SELECT_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_CLEAR_COUNTERS_OFFS		4
-#define MV_PTP_GENERAL_CTRL_CLEAR_COUNTERS_MASK    \
-		(0x00000001 << MV_PTP_GENERAL_CTRL_CLEAR_COUNTERS_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_TAI_SELECT_OFFS		5
-#define MV_PTP_GENERAL_CTRL_TAI_SELECT_MASK    \
-		(0x00000001 << MV_PTP_GENERAL_CTRL_TAI_SELECT_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_TS_QUEUE_OVER_WRITE_ENABLE_OFFS		6
-#define MV_PTP_GENERAL_CTRL_TS_QUEUE_OVER_WRITE_ENABLE_MASK    \
-	(0x00000001 << MV_PTP_GENERAL_CTRL_TS_QUEUE_OVER_WRITE_ENABLE_OFFS)
-
-#define MV_PTP_GENERAL_CTRL_TAI_ACK_DELAY_OFFS		7
-#define MV_PTP_GENERAL_CTRL_TAI_ACK_DELAY_MASK    \
-		(0x0000001f << MV_PTP_GENERAL_CTRL_TAI_ACK_DELAY_OFFS)
-
-/* Ptp Tx Timestamp Queue0 Reg0 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_REG				(0x080c)
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_PTP_TX_TIMESTAMP_QUEUE0_VALID_OFFS 0
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_PTP_TX_TIMESTAMP_QUEUE0_VALID_MASK    \
-	(0x00000001 << \
-	MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_PTP_TX_TIMESTAMP_QUEUE0_VALID_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_QUEUE_ID_OFFS		1
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_QUEUE_ID_MASK    \
-		(0x000003ff << MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_QUEUE_ID_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TAI_SELECT_OFFS		11
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TAI_SELECT_MASK    \
-		(0x00000001 << MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TAI_SELECT_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TOD_UPDATE_FLAG_OFFS		12
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TOD_UPDATE_FLAG_MASK    \
-	(0x00000001 << MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TOD_UPDATE_FLAG_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TIMESTAMP_BITS_0_2_OFFS		13
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TIMESTAMP_BITS_0_2_MASK    \
-	(0x00000007 << MV_PTP_TX_TIMESTAMP_QUEUE0_REG0_TIMESTAMP_BITS_0_2_OFFS)
-
-/* Ptp Tx Timestamp Queue0 Reg1 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG1_REG				(0x0810)
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG1_TIMESTAMP_BITS_3_18_OFFS	0
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG1_TIMESTAMP_BITS_3_18_MASK    \
-	(0x0000ffff << \
-	MV_PTP_TX_TIMESTAMP_QUEUE0_REG1_TIMESTAMP_BITS_3_18_OFFS)
-
-/* Ptp Tx Timestamp Queue0 Reg2 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG2_REG				(0x0814)
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG2_TIMESTAMP_BITS_19_31_OFFS	0
-#define MV_PTP_TX_TIMESTAMP_QUEUE0_REG2_TIMESTAMP_BITS_19_31_MASK    \
-	(0x00001fff << \
-	MV_PTP_TX_TIMESTAMP_QUEUE0_REG2_TIMESTAMP_BITS_19_31_OFFS)
-
-/* Ptp Tx Timestamp Queue1 Reg0 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_REG				(0x0818)
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_PTP_TX_TIMESTAMP_QUEUE1_VALID_OFFS 0
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_PTP_TX_TIMESTAMP_QUEUE1_VALID_MASK    \
-	(0x00000001 << \
-	MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_PTP_TX_TIMESTAMP_QUEUE1_VALID_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_QUEUE_ID_OFFS		1
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_QUEUE_ID_MASK    \
-		(0x000003ff << MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_QUEUE_ID_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TAI_SELECT_OFFS		11
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TAI_SELECT_MASK    \
-		(0x00000001 << MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TAI_SELECT_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TOD_UPDATE_FLAG_OFFS		12
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TOD_UPDATE_FLAG_MASK    \
-	(0x00000001 << MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TOD_UPDATE_FLAG_OFFS)
-
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TIMESTAMP_BITS_0_2_OFFS		13
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TIMESTAMP_BITS_0_2_MASK    \
-	(0x00000007 << MV_PTP_TX_TIMESTAMP_QUEUE1_REG0_TIMESTAMP_BITS_0_2_OFFS)
-
-/* Ptp Tx Timestamp Queue1 Reg1 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG1_REG				(0x081c)
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG1_TIMESTAMP_BITS_3_18_OFFS	0
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG1_TIMESTAMP_BITS_3_18_MASK    \
-	(0x0000ffff << MV_PTP_TX_TIMESTAMP_QUEUE1_REG1_TIMESTAMP_BITS_3_18_OFFS)
-
-/* Ptp Tx Timestamp Queue1 Reg2 */
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG2_REG				(0x0820)
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG2_TIMESTAMP_BITS_19_31_OFFS	0
-#define MV_PTP_TX_TIMESTAMP_QUEUE1_REG2_TIMESTAMP_BITS_19_31_MASK    \
-	(0x00001fff << \
-	MV_PTP_TX_TIMESTAMP_QUEUE1_REG2_TIMESTAMP_BITS_19_31_OFFS)
-
-/* Total Ptp Packets Counter */
-#define MV_PTP_TOTAL_PTP_PCKTS_CNTR_REG				(0x0824)
-#define MV_PTP_TOTAL_PTP_PCKTS_CNTR_TOTAL_PTP_PACKETS_COUNTER_OFFS	0
-#define MV_PTP_TOTAL_PTP_PCKTS_CNTR_TOTAL_PTP_PACKETS_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_TOTAL_PTP_PCKTS_CNTR_TOTAL_PTP_PACKETS_COUNTER_OFFS)
-
-/* Ptpv1 Packet Counter */
-#define MV_PTP_PTPV1_PCKT_CNTR_REG				(0x0828)
-#define MV_PTP_PTPV1_PCKT_CNTR_PTPV1_PACKET_COUNTER_OFFS		0
-#define MV_PTP_PTPV1_PCKT_CNTR_PTPV1_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_PTPV1_PCKT_CNTR_PTPV1_PACKET_COUNTER_OFFS)
-
-/* Ptpv2 Packet Counter */
-#define MV_PTP_PTPV2_PCKT_CNTR_REG				(0x082c)
-#define MV_PTP_PTPV2_PCKT_CNTR_PTPV2_PACKET_COUNTER_OFFS		0
-#define MV_PTP_PTPV2_PCKT_CNTR_PTPV2_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_PTPV2_PCKT_CNTR_PTPV2_PACKET_COUNTER_OFFS)
-
-/* Y1731 Packet Counter */
-#define MV_PTP_Y1731_PCKT_CNTR_REG				(0x0830)
-#define MV_PTP_Y1731_PCKT_CNTR_Y1731_PACKET_COUNTER_OFFS		0
-#define MV_PTP_Y1731_PCKT_CNTR_Y1731_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_Y1731_PCKT_CNTR_Y1731_PACKET_COUNTER_OFFS)
-
-/* Ntpts Packet Counter */
-#define MV_PTP_NTPTS_PCKT_CNTR_REG				(0x0834)
-#define MV_PTP_NTPTS_PCKT_CNTR_NTPTS_PACKET_COUNTER_OFFS		0
-#define MV_PTP_NTPTS_PCKT_CNTR_NTPTS_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_NTPTS_PCKT_CNTR_NTPTS_PACKET_COUNTER_OFFS)
-
-/* Ntpreceive Packet Counter */
-#define MV_PTP_NTPRECEIVE_PCKT_CNTR_REG				(0x0838)
-#define MV_PTP_NTPRECEIVE_PCKT_CNTR_NTPRX_PACKET_COUNTER_OFFS		0
-#define MV_PTP_NTPRECEIVE_PCKT_CNTR_NTPRX_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_NTPRECEIVE_PCKT_CNTR_NTPRX_PACKET_COUNTER_OFFS)
-
-/* Ntptransmit Packet Counter */
-#define MV_PTP_NTPTRANSMIT_PCKT_CNTR_REG			(0x083c)
-#define MV_PTP_NTPTRANSMIT_PCKT_CNTR_NTPTX_PACKET_COUNTER_OFFS		0
-#define MV_PTP_NTPTRANSMIT_PCKT_CNTR_NTPTX_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_NTPTRANSMIT_PCKT_CNTR_NTPTX_PACKET_COUNTER_OFFS)
-
-/* Wamp Packet Counter */
-#define MV_PTP_WAMP_PCKT_CNTR_REG				(0x0840)
-#define MV_PTP_WAMP_PCKT_CNTR_WAMP_PACKET_COUNTER_OFFS		0
-#define MV_PTP_WAMP_PCKT_CNTR_WAMP_PACKET_COUNTER_MASK    \
-		(0x000000ff << MV_PTP_WAMP_PCKT_CNTR_WAMP_PACKET_COUNTER_OFFS)
-
-/* None Action Packet Counter */
-#define MV_PTP_NONE_ACTION_PCKT_CNTR_REG			(0x0844)
-#define MV_PTP_NONE_ACTION_PCKT_CNTR_NONE_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_NONE_ACTION_PCKT_CNTR_NONE_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_NONE_ACTION_PCKT_CNTR_NONE_ACTION_PACKET_COUNTER_OFFS)
-
-/* Forward Action Packet Counter */
-#define MV_PTP_FORWARD_ACTION_PCKT_CNTR_REG			(0x0848)
-#define MV_PTP_FORWARD_ACTION_PCKT_CNTR_FORWARD_ACTION_PACKET_COUNTER_OFFS 0
-#define MV_PTP_FORWARD_ACTION_PCKT_CNTR_FORWARD_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_FORWARD_ACTION_PCKT_CNTR_FORWARD_ACTION_PACKET_COUNTER_OFFS)
-
-/* Drop Action Packet Counter */
-#define MV_PTP_DROP_ACTION_PCKT_CNTR_REG			(0x084c)
-#define MV_PTP_DROP_ACTION_PCKT_CNTR_DROP_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_DROP_ACTION_PCKT_CNTR_DROP_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_DROP_ACTION_PCKT_CNTR_DROP_ACTION_PACKET_COUNTER_OFFS)
-
-/* Capture Action Packet Counter */
-#define MV_PTP_CAPTURE_ACTION_PCKT_CNTR_REG			(0x0850)
-#define MV_PTP_CAPTURE_ACTION_PCKT_CNTR_CAPTURE_ACTION_PACKET_COUNTER_OFFS 0
-#define MV_PTP_CAPTURE_ACTION_PCKT_CNTR_CAPTURE_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_CAPTURE_ACTION_PCKT_CNTR_CAPTURE_ACTION_PACKET_COUNTER_OFFS)
-
-/* Addtime Action Packet Counter */
-#define MV_PTP_ADDTIME_ACTION_PCKT_CNTR_REG			(0x0854)
-#define MV_PTP_ADDTIME_ACTION_PCKT_CNTR_ADDTIME_ACTION_PACKET_COUNTER_OFFS 0
-#define MV_PTP_ADDTIME_ACTION_PCKT_CNTR_ADDTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_ADDTIME_ACTION_PCKT_CNTR_ADDTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Addcorrectedtime Action Packet Counter */
-#define MV_PTP_ADDCORRECTEDTIME_ACTION_PCKT_CNTR_REG		(0x0858)
-#define MV_PTP_ADDCORRECTEDTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_ADDCORRECTEDTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_ADDCORRECTEDTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Captureaddtime Action Packet Counter */
-#define MV_PTP_CAPTUREADDTIME_ACTION_PCKT_CNTR_REG		(0x085c)
-#define MV_PTP_CAPTUREADDTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_CAPTUREADDTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_CAPTUREADDTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Captureaddcorrectedtime Action Packet Counter */
-#define MV_PTP_CAPTUREADDCORRECTEDTIME_ACTION_PCKT_CNTR_REG	(0x0860)
-#define MV_PTP_CAPTUREADDCORRECTEDTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_CAPTUREADDCORRECTEDTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << \
-	MV_PTP_CAPTUREADDCORRECTEDTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Addingresstime Action Packet Counter */
-#define MV_PTP_ADDINGRESSTIME_ACTION_PCKT_CNTR_REG		(0x0864)
-#define MV_PTP_ADDINGRESSTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_ADDINGRESSTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_ADDINGRESSTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Captureaddingresstime Action Packet Counter */
-#define MV_PTP_CAPTUREADDINGRESSTIME_ACTION_PCKT_CNTR_REG	(0x0868)
-#define MV_PTP_CAPTUREADDINGRESSTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_CAPTUREADDINGRESSTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_CAPTUREADDINGRESSTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Captureingresstime Action Packet Counter */
-#define MV_PTP_CAPTUREINGRESSTIME_ACTION_PCKT_CNTR_REG		(0x086c)
-#define MV_PTP_CAPTUREINGRESSTIME_ACTION_PACKET_COUNTER_OFFS	0
-#define MV_PTP_CAPTUREINGRESSTIME_ACTION_PACKET_COUNTER_MASK    \
-	(0x000000ff << MV_PTP_CAPTUREINGRESSTIME_ACTION_PACKET_COUNTER_OFFS)
-
-/* Ntp Ptp Offset High */
-#define MV_PTP_NTP_PTP_OFFSET_HIGH_REG				(0x0870)
-#define MV_PTP_NTP_PTP_OFFSET_HIGH_PTP_NTP_OFFSET_HIGH_OFFS	0
-#define MV_PTP_NTP_PTP_OFFSET_HIGH_PTP_NTP_OFFSET_HIGH_MASK    \
-	(0x0000ffff << MV_PTP_NTP_PTP_OFFSET_HIGH_PTP_NTP_OFFSET_HIGH_OFFS)
-
-/* Ntp Ptp Offset Low */
-#define MV_PTP_NTP_PTP_OFFSET_LOW_REG				(0x0874)
-#define MV_PTP_NTP_PTP_OFFSET_LOW_PTP_NTP_OFFSET_LOW_OFFS	0
-#define MV_PTP_NTP_PTP_OFFSET_LOW_PTP_NTP_OFFSET_LOW_MASK    \
-	(0x0000ffff << MV_PTP_NTP_PTP_OFFSET_LOW_PTP_NTP_OFFSET_LOW_OFFS)
 
 /******************************************************************************/
 /* System Soft Reset 1 */
-- 
1.7.9.5

